src/data/automation_i18n.ts

```diff
@@ -188,15 +188,16 @@ export const describeTrigger = (
 
   // Time Trigger
   if (trigger.platform === "time" && trigger.at) {
-    const at = trigger.at.includes(".")
-      ? `entity ${
-          hass.states[trigger.at]
-            ? computeStateName(hass.states[trigger.at])
-            : trigger.at
-        }`
-      : trigger.at;
-
-    return `When the time is equal to ${at}`;
+    const result = ensureArray(trigger.at).map((at) =>
+      at.toString().includes(".")
+        ? `entity ${hass.states[at] ? computeStateName(hass.states[at]) : at}`
+        : at
+    );
+
+    const last = result.splice(-1, 1)[0];
+    return `When the time is equal to ${
```

Comment:
Should this be localized?

Reply:
Nothing is the automation/scripts i18n is localized yet, as we first want to finalize the format is english.

Reply:
I think we are almost there btw